onnxruntime/test/python/onnxruntime_test_ort_trainer.py

```diff
@@ -313,5 +329,63 @@ def testBertTrainingGradientAccumulation(self):
         assert_allclose(expected_losses, actual_losses, err_msg="loss mismatch")
         assert_allclose(expected_eval_loss, actual_eval_loss, err_msg="evaluation loss mismatch")
 
+    def testBertCheckpointingBasic(self):
+        model,_,_ = create_ort_trainer(gradient_accumulation_steps=1,
+                        use_mixed_precision=False,
+                        allreduce_post_accumulation=True,
+                        use_simple_model_desc=True,
+                        loss_scaler=None)
+        sd = model.state_dict()
+
+        # modify one of the default values
+        sd['bert.encoder.layer.0.attention.output.LayerNorm.weight'] +=1
+        model.load_state_dict(sd)
+
+        ckpt_dir = get_name("ort_ckpt")
+        model.save_checkpoint(ckpt_dir, 'bert_toy_save')
+        del model
+
+        # create new model
+        model2,_,_ = create_ort_trainer(gradient_accumulation_steps=1,
+                        use_mixed_precision=False,
+                        allreduce_post_accumulation=True,
+                        use_simple_model_desc=True,
+                        loss_scaler=None)
+
+        # load changed checkpoint
+        model2.load_checkpoint('./', 'bert_toy_save')
+        loaded_sd = model2.state_dict()
+
+        for k,v in loaded_sd.items():
+            assert torch.all(torch.eq(v, sd[k]))
+
+    def testBertCheckpointingLoadZero(self):
+        torch.manual_seed(1)
+        onnxruntime.set_seed(1)
+        model,_,device = create_ort_trainer(gradient_accumulation_steps=1,
+                        use_mixed_precision=False,
+                        allreduce_post_accumulation=True,
+                        use_simple_model_desc=True,
+                        loss_scaler=None)
+
+        ckpt_dir = get_name("ort_ckpt")
+        model.load_checkpoint(ckpt_dir, 'bert_toy_lamb_', is_partitioned=True)
```

Comment:
Is it possible to add a unit test that creates a ZeRO checkpoint, then loads it?  Rather than committing the checkpoint files?